sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultAccessControlAsyncClient.java

```diff
@@ -1044,4 +1045,26 @@ public T getValue() {
             return output;
         }
     }
+
+    /**
+     * Deserializes a given {@link Response HTTP response} including headers to a given class.
+     *
+     * @param statusCode The status code which will trigger exception swallowing.
+     * @param httpResponseException The {@link HttpResponseException} to be swallowed.
+     * @param logger {@link ClientLogger} that will be used to record the exception.
+     * @param <E> The class of the exception to swallow.
+     *
+     * @return A {@link Mono} that contains the deserialized response.
+     */
+    static <E extends HttpResponseException> Mono<Response<Void>> swallowExceptionForStatusCode(int statusCode,
```

Comment:
Did you not copy my transforms I posted in Teams? It would be better to not throw than to throw and swallow an exception, would it not?

Reply:
I had this code already written before I saw your transforms for the swagger. I can add them and re-generate :)